coalib/output/Logging.py

```diff
@@ -33,3 +36,64 @@ def configure_logging():
             }
         }
     })
+
+
+def configure_json_logging():
+    """
+    Configures logging as a JSON objects.
+    Returns a StringIO() that captures the logs as json.
+    """
+    import logging.config
+    import io
+
+    stream = io.StringIO()
+
+    logging.config.dictConfig({
+        'version': 1,
+        'handlers': {
+            'json': {
+                'class': 'logging.StreamHandler',
+                'formatter': 'json',
+                'stream': stream
+            }
+        },
+        'root': {
+            'level': 'DEBUG',
+            'handlers': ['json']
+        },
+        'formatters': {
+            'json': {
+                '()': 'coalib.output.Logging.JSONFormatters',
+            }
+        }
+    })
+    return stream
+
+
+class JSONFormatters(logging.Formatter):
+    """
+    JSON formatter for python logging
+    """
+
+    def __init__(self):
+        pass
+
+    @classmethod
+    def format_timestamp(cls, time):
+        from datetime import datetime
+
+        return datetime.utcfromtimestamp(time).isoformat() + 'Z'
+
+    def format(self, record, serialize=True):
```

Comment:
Heres what I understood. A staticmethod function is some function that does not require `self` (instance of an class). A classmethod function is some function that requires `cls` (the class itself).

Here since we dont need `cls` as we are not using any other functions/variables that are defined in the class or its parent classes we wont use classmethod funtion.

And since we also dont need instance functions/variables we will use a staticmethod funtion.